src/pages/components/box.mdx

```diff
@@ -5,6 +5,100 @@ layout: '@layouts/ComponentLayout.astro'
 description: 'Webレイアウトの基本であるボックスを提供します'
 thumbnail: '/assets/images/components/thumbnail-box.svg'
 repositoryUrl: 'https://github.com/ubie-oss/ubie-ui/tree/main/src/components/Box'
+props:
+- name: "children"
+  description: "Box内に表示するコンテンツやコンポーネント"
+  type: "ReactNode"
+- name: "as"
+  description: "レンダリングされるHTML要素"
+  defaultValue: "div"
+  type: "'HTMLの要素名'"
+- name: "backgroundColor"
+  description: "背景色"
+  type: "{BackgroundColor}"
```

Comment:
It is hard to write the same props every time. To be able to specify keywords so that they can be used over and over again.